ipatests/test_integration/test_ipa_idrange_fix.py

```diff
@@ -20,6 +20,7 @@ class TestIpaIdrangeFix(IntegrationTest):
     @classmethod
     def install(cls, mh):
         super(TestIpaIdrangeFix, cls).install(mh)
+        tasks.install_master(cls.master, setup_dns=True)
         tasks.kinit_admin(cls.master)
```

Comment:
super(TestIpaIdrangeFix, cls).install(mh) should do the installation but it doesn't because no topology is defined in the class. If that is defined the server should be installed.